cmd/oras/internal/display/content/interface.go

```diff
@@ -31,5 +31,11 @@ type ManifestIndexCreateHandler interface {
 	OnContentCreated(content []byte) error
 }
 
+// BlobPushHandler handles descriptor output for blob push events.
+type BlobPushHandler interface {
+	// OnBlobPushed is called after a blob is pushed.
+	OnBlobPushed() error
```

Comment:
by logic, it should be
```suggestion
	OnBlobPushed(desc ocispec.Descriptor) error
```